lib/src/models/ton_wallet_related/ton_wallet_multisig_pending_transaction.dart

```diff
@@ -14,21 +14,26 @@ class TonWalletMultisigPendingTransaction
     required PublicKey creator,
     // custodians who accepted this transaction
     required List<PublicKey> confirmations,
-    // list of possible custodians
+    // list of possible custodians for account
     required List<PublicKey> custodians,
     // if this transaction was sent from this wallet
     required bool isOutgoing,
     @amountJsonConverter required BigInt value,
     // address of destination or source depends on isOutgoing
     required Address address,
+    // address of account for which this transaction was found
+    required Address walletAddress,
     required DateTime date,
     @amountJsonConverter required BigInt fees,
     required String hash,
     required int signsReceived,
     required int signsRequired,
     required String transactionId,
-    required List<PublicKey> publicKeys,
+    // local custodians that can confirm transaction
```

Comment:
Based on field name it's more like list of custodians who have not confirm the transaction yet.

Reply:
It's the same. Not confirmed or can confirm